src/index.html

```diff
@@ -11,6 +11,45 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>Search bar airbnb</h1>
+    <form
+    method="get"
+    action="/"
+    data-qa="big"
+    >
```

Comment:
```suggestion
    <form
      method="get"
      action="/"
      data-qa="big"
    >
```

fix all simillar cases